src/modal.js

```diff
@@ -82,15 +81,15 @@ function modal ({
     }
 
     const connectedModalWrapper = connectModal({
-      name: modalName,
+      name,
       ...options,
     })(ModalWrapper)
-    // Rename the whole thing to the modal name so we can call actions on it`
-    connectedModalWrapper.displayName = modalName
+    // Rename the whole thing to the modal name so we can call actions on it
+    connectedModalWrapper.displayName = name
```

Comment:
This line is no longer really relevant to how we do modals, might be more confusing than useful.

We used to call `show(Modal.displayName)` to show modals, now we call `Modal.show()` instead.